README.md

```diff
@@ -236,6 +238,33 @@ $client = new Client(
 );
 ```
 
+
+The Client constructor also receives an optional "httpOptions" array, which can be used to add custom [Guzzle HTTP Client request options](https://guzzle.readthedocs.io/en/latest/request-options.html).
```

Comment:
@actionm Please add to the docs that the httpOptions argument value overrides the authorizationHeaders